src/core/ext/xds/xds_client.cc

```diff
@@ -46,6 +46,7 @@
 #include "src/core/lib/gprpp/orphanable.h"
 #include "src/core/lib/gprpp/ref_counted_ptr.h"
 #include "src/core/lib/gprpp/sync.h"
+#include "src/core/lib/iomgr/exec_ctx.h"
```

Comment:
It's odd that we're adding this include here, when we're removing references to `ExecCtx`.  We removed this include in all of the other files.  Any idea why this one is different?

Reply:
I do not... happy to dig a little bit (happy to not spend the time too depending on how strongly you feel)

Reply:
I guess I don't care too much, since it's not really a new dependency.  If you can't figure it out quickly, it's probably not worth spending time on.

Reply:
so it's there for the `ApplicationExecCtx` and `ExecCtx` usages within this file - this is right...
I didn't get an answer to why it doesn't currently exist.

Reply:
Oh I have that answer too: iwyu treats the header & implementation file as a set, so the removal of exec_ctx.h from the header whilst there's still usage in the implementation file means that the implementation file needs to pick up exec_ctx.h.

Reply:
Ah!  That makes sense.  Thanks!